notebooks/point-cloud-processing.ipynb

```diff
@@ -2,9 +2,37 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 1,
```

Comment:
could you clear all output of the notebook so as to make diffs cleaner?

Reply:
ops... sorry